src/Plugin/Field/FieldType/OgGroupItem.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\og\OgGroupItem.
+ */
+
+namespace Drupal\og\Plugin\Field\FieldType;
+
+use Drupal\Core\Field\FieldItemBase;
+use Drupal\Core\Field\FieldStorageDefinitionInterface;
+
+/**
+ * OG Group Data class to return TRUE always.
+ *
+ * @FieldType(
```

Comment:
`// This field type is used simply as a placeholder for a field formatter that would allow non-groups members, members, and group managers to join, request or leave the a group.`